crud-operations/src/main/java/org/example/crudoperations/service/ProductService.java

```diff
@@ -0,0 +1,50 @@
+package org.example.crudoperations.service;
+
+import org.example.crudoperations.entity.Product;
+import org.example.crudoperations.repository.ProductRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+public class ProductService {
+    @Autowired
+    private ProductRepository repository;
```

Comment:
Say we add another repository as dependency. Then the name `repository` doesn't make much sense right. Name it appropriately.